guide/popular-topics/builders.md

```diff
@@ -0,0 +1,132 @@
+# Builders
+
+Discord.js provides a `builders` package which contains a variety of builders and utilities you can use when writing your Discord bot.
+To install the package run `npm i @discordjs/builders` in your terminal.
+
+## Formatters
+
+Formatters are a set of utility functions which format input strings into the given format.
+
+### Basic Markdown
+
+The Formatters provide functions to format strings into all the different Markdown styles supported by Discord.
+
+```js
+const { Formatters } = require('@discordjs/builders');
+const string = 'Hello!';
+
+const bold = Formatters.bold(string);
+const italic = Formatters.italic(string);
+const strikethrough = Formatters.strikethrough(string);
+const spoiler = Formatters.spoiler(string);
+const quote = Formatters.quote(string);
+const blockquote = Formatters.blockQuote(string);
+const underscore = Formatters.underscore(string);
```

Comment:
There is no `Formatters` class, you need to export each function you'd like to use individually ([eg](https://github.com/discordjs/builders/blob/main/tests/messages.test.ts#L1-L20))